tests/openassetio/test/managerValidator/conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
```

Comment:
[fix] If we keep this file, needs license header.

Reply:
Added license headers in db502f8